backend/manager/modules/bll/src/test/java/org/ovirt/engine/core/bll/scheduling/utils/VdsCpuUnitPinningHelperTest.java

```diff
@@ -416,4 +416,41 @@ public void shouldSucceedToDedicateEqualSockets() {
         assertEquals(6, cpus.stream().filter(cpu -> cpu.getSocket() == 1).count());
     }
 
+    @Test
+    public void releaseIsolatedThreads() {
+        // 2:3:1 vm takes 12 CPUs, while allocating we need to drop one the last core in socket 1.
```

Comment:
can you please rephrase? I understand the first part that vm with cpu topology 2:3:1 and isolated threads policy takes 12 CPUs on a host with 2 threads-per-core but the last part is not clear (here, from a black box point of view it doesn't make sense that we would over-allocate cpus for a vm with 2:3:1 on a host with 2:4:2 - if it's hard to justify that, we can drop that comment)